test/alternator/test_describe_table.py

```diff
@@ -112,12 +120,73 @@ def test_describe_table_item_count(test_table):
     got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
     assert 'ItemCount' in got
 
-# Similar test for estimated size in bytes - TableSizeBytes - which again,
-# may reflect the size as long as six hours ago.
-def test_describe_table_size(test_table):
-    got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
-    assert 'TableSizeBytes' in got
-    assert got['TableSizeBytes'] >= 0
+def test_describe_table_size_with_timeout_set_to_0(cql, dynamodb):
+    # UPDATE system.config SET value = 0 WHERE name = alternator_describe_table_info_timeout_in_seconds
+
+    schema = {
+        'KeySchema': [ { 'AttributeName': 'p', 'KeyType': 'HASH' },
+                    { 'AttributeName': 'c', 'KeyType': 'RANGE' }
+        ],
+        'AttributeDefinitions': [
+                    { 'AttributeName': 'p', 'AttributeType': 'S' },
+                    { 'AttributeName': 'c', 'AttributeType': 'S' },
+        ],
+    }
+
+    with config_value_context(cql, 'alternator_describe_table_info_timeout_in_seconds', '0'):
+        with new_test_table(dynamodb, **schema) as test_table:
+            got1 = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
+            assert 'TableSizeBytes' in got1
+            assert got1['TableSizeBytes'] >= 0, (got1['TableSizeBytes'], )
+
+            for x in range(0, 128):
+                p = random_string()
+                c = random_string()
+                v = random_string()
+                test_table.put_item(Item={'p': p, 'c': c, 'v': v})
```

Comment:
Sending 128 writes takes 128 times more time than one write. Why isn't one write enough?
We're not stress testing PutItem here. You just wanted to have some sstables.